spec/factories/organizations.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :organization do
+    name 'test-org'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.